### ISSUE TITLE
Manager not displayed in newer version of Chrome

### ISSUE THREAD

**Issue by polnops** [labrad/scalabrad-web#331]
When I start the manager and run it on a new version of Chrome (e.g. after 80.0), the manager fails to display (even though my servers still run correctly).  

Reverting Chrome to v55.0 fixes the problem, but I'm just curious if anyone has successfully had the manager show up correctly in the newest version of Chrome?

**Comment by maffoo** [comment]
This is a known issue. Chrome 80 dropped support for html imports, which are needed by the polymer library used in scalabrad-web. Fixing this would require migrating to polymer v3, which is a pretty big chunk of work.

**Comment by jayich** [comment]
https://www.chromestatus.com/feature/5144752345317376

**Comment by gschaffner** [comment]
Hi @maffoo --- I'm attempting to gauge how much work this would entail and how feasible it would be for someone who is unfamiliar with Node (me) to take this on. So far, my working understanding is that the path would be something like

*   [x] Replicate a build of the current master and verify that it works on Chromium < 80.
*   [ ] [Upgrade from Polymer 1.6 to 2.0](https://polymer-library.polymer-project.org/2.0/docs/upgrade).
    *   [x] Get `PolymerElements/{iron,neon,paper}-elements` 2.0 as individual elements rather than as a pack. The `{iron,neon,paper}-elements` repos haven't been upgraded for Polymer 2.0. See [here](https://stackoverflow.com/questions/44736250/polymer-2-migration-issues).
        *    [x] [paper-menu](https://github.com/PolymerElements/paper-menu) ends with v1 and is deprecated in favor of paper-listbox. We need to migrate unless we want to just use the unmaintained fork PolymerElements/paper-menu#113.
    *   [ ] Migrate from PolymerTS to PolymerDecorators. This might be partially automated: see nippur72/PolymerTS#110.
    *   [ ] ??
*   [ ] [Upgrade from Polymer 2.0 to 3.0](https://polymer-library.polymer-project.org/3.0/docs/upgrade).
    *   [ ] This upgrade is mostly automated by Polymer modulizer.
    *   [ ] ??

Does this look accurate to you? Any large steps that I'm missing in the `??` areas? Thanks!

**Comment by fanmingyu212** [comment]
A temporary workaround for chrome version > 80: https://github.com/karma-runner/karma/issues/3419#issuecomment-583138545. I think this issue should still be addressed if possible though.

**Comment by xiaoyuejin** [comment]
Will this issue be further addressed?

**Comment by gschaffner** [comment]
I'm hoping to continue working on this, but I'm completely unfamiliar with web tech and was hoping for some advice from @maffoo before continuing.

Current progress is [here](https://github.com/labrad/scalabrad-web/tree/u/gschaffner/polymer-2.0).

**Comment by gschaffner** [comment]
@maffoo Are you working on this? I saw your new branch and wanted to make sure you'd seen this issue and [this branch](https://github.com/labrad/scalabrad-web/tree/u/gschaffner/polymer-2.0). If you are, it'd be great if you could push as you go as I'm interested in reading through the changes that are made to upgrade.

**Comment by xiaoyuejin** [comment]
Just wondering: Is anyone still trying to solve this issue? Actually, my question is, how to go around this problem? I mean there must be a way to do that. I can't imagine the entire google team is using an old version of Chrome on all the measurement computers...

**Comment by fanmingyu212** [comment]
@xiaoyuejin see the workaround above: https://github.com/labrad/scalabrad-web/issues/331#issuecomment-601955326. This still works for us.

**Comment by fanmingyu212** [comment]
It seems that HTMLImports was completely removed from chrome in version 88. The workaround above does not work anymore. The solution we use right now is to keep chrome up to date (it does not work with labrad), and install another chromium-based browser that is not updated so it would work with labrad.

**Comment by xiaoyuejin** [comment]
Yeah, that's exactly why I was asking the questions I asked above...
